BraveRewardsUI/Wallet/Notifications/WalletNotificationView.swift

```diff
@@ -23,21 +70,72 @@ class WalletNotificationView: UIView {
     
     addSubview(backgroundView)
     addSubview(closeButton)
+    addSubview(stackView)
+    stackView.addArrangedSubview(iconImageView)
     
     closeButton.snp.makeConstraints {
       $0.top.trailing.equalTo(safeAreaLayoutGuide)
       $0.width.height.equalTo(44.0)
     }
+    
+    stackView.snp.makeConstraints {
+      $0.top.greaterThanOrEqualTo(self).offset(15.0)
+      $0.centerY.equalToSuperview()
+      $0.leading.trailing.equalTo(safeAreaLayoutGuide).inset(35.0)
+      $0.bottom.lessThanOrEqualTo(self).inset(25.0)
+    }
   }
   
   override func layoutSubviews() {
     super.layoutSubviews()
-    
     backgroundView.frame = bounds
   }
   
   @available(*, unavailable)
   required init(coder: NSCoder) {
     fatalError()
   }
+  
+  private func bodyAttributedString() -> NSAttributedString {
+    guard let notification = notification else {
+      return NSAttributedString()
```

Comment:
add `assertionFailure` here

Reply:
No more required after creating a separate class for message type